test/lib/commands/install.js

```diff
@@ -687,4 +687,33 @@ t.test('devEngines', async t => {
     t.ok(output.includes('EBADENGINE'))
     t.ok(!output.includes('EBADDEVENGINES'))
   })
+
+  t.test('should not utilize devEngines if global install', async t => {
+    const { npm, joinedFullOutput } = await loadMockNpm(t, {
+      ...mockArguments,
+      config: {
+        global: true,
+      },
```

Comment:
nitty nit nit: I've noticed config is typically one of the last keys in this test object